src/s3ql/mount.py

```diff
@@ -598,6 +598,9 @@ async def run(self):
         log.debug('started')
 
         assert self.fs is not None
+        if self.interval == None:
+            log.info('Metadata upload disabled.')
+            self.quit = True
```

Comment:
Looking at https://github.com/s3ql/s3ql/blob/master/src/s3ql/fs.py#L256, wouldn't this break metadata upload with s3qlctrl?

Reply:
I've been looking at this most of the morning and trying things out (without luck). I will continue to play with it, but a lot of this is foreign to me.  If there are any suggestions on bypassing periodic metadata uploads but still allowing on-demand ones, I'd be happy to hear them.  Its probably a simple solution but I'm not up to speed on how all of this works together.

Reply:
I think the right solution is to call just `await self.event.wait()` (without wrapping it into `trio.move_on_after`) if periodic uploads are disabled.

Reply:
Thanks for the response!  I was actually just updating the code to try that out.  :)